docs/onboarding.md

```diff
@@ -33,7 +33,9 @@ between different product types:
   [Docker image](docker.md)
 * **Service Product** to start [onboarding a new service](service.md), that is
   not tied to the distribution of an application to the end-user
-* **API Product** to onboard a new [API-based service](api-product.md)
+* **API Product** to onboard a new [API-based service](api-product.md). This
+  option is available only on environments where the *API calls reselling*
+  capability via Kong is enabeld
```

Comment:
tutti i product types sono dietro feature toggles, se ha senso specificarlo va fatto a monte per tutti

Reply:
hai ragione, ho aggiunto ulteriori commenti

Reply:
typo: via Kong is enabeld

Reply:
@ipasqua-eng stavi vedendo la versione vecchia